.github/CODEOWNERS

```
@@ -123,16 +123,9 @@ ui/pages/home                         @MetaMask/wallet-ux
 ui/pages/onboarding-flow              @MetaMask/wallet-ux
 
 # Assets
-ui/components/app/add-network                         @MetaMask/metamask-assets
-ui/components/app/auto-detect-nft                     @MetaMask/metamask-assets
-ui/components/app/auto-detect-token                   @MetaMask/metamask-assets
+ui/components/app/detected-token                   @MetaMask/metamask-assets
 ui/components/app/import-token                        @MetaMask/metamask-assets
-ui/components/app/nft-default-image                   @MetaMask/metamask-assets
-ui/components/app/nft-details                         @MetaMask/metamask-assets
-ui/components/app/nft-options                         @MetaMask/metamask-assets
-ui/components/app/nfts-detection-notice-import-nfts   @MetaMask/metamask-assets
-ui/components/app/nfts-items                          @MetaMask/metamask-assets
-ui/components/app/nfts-tab                            @MetaMask/metamask-assets
+ui/components/app/nfts                                @MetaMask/metamask-assets
```

Looks like this directory does not exist. It should be `ui/components/app/**assets**/nfts`.

```suggestion
ui/components/app/assets                              @MetaMask/metamask-assets
```

---

@Gudahtt Thanks again😅

@MetaMask/metamask-assets It looks like everything under `ui/components/app/assets` should be owned by you guys, not just the `nfts` subdir. Could I get a confirmation of this?